panels/user-accounts/um-account-dialog.c

```diff
@@ -1678,3 +1683,21 @@ um_account_dialog_finish (UmAccountDialog     *self,
         g_clear_object (&self->async);
         return user;
 }
+
+void
+um_account_dialog_set_is_small_screen (UmAccountDialog *self,
+                                       gboolean         is_small_screen)
+{
+        if (is_small_screen) {
```

Comment:
The reason I was asking is because the other similar functions you add for the dialogs take both cases into account.
I guess if we never expect this to change during runtime, those other cases can be made simpler too?